argo-cd-apps/overlays/production-downstream/kustomization.yaml

```diff
@@ -182,3 +182,8 @@ patches:
       kind: ApplicationSet
       version: v1alpha1
       name: spacerequest-cleaner
+  - path: production-overlay-patch.yaml
```

Comment:
why is it needed?

Reply:
See my other comment.

Reply:
Updated. The eaas base is now removed so there's no need for a delete patch.